core/src/main/scala/org/apache/spark/storage/StorageLevel.scala

```diff
@@ -132,6 +132,10 @@ class StorageLevel private(
 
   override def hashCode(): Int = toInt * 41 + replication
 
+  /** Name of the storage level if it is predefined or `None` otherwise. */
+  def name: Option[String] = StorageLevel.PREDEFINED
+    .collectFirst { case (storageLevel, name) if storageLevel == this => name }
```

Comment:
If the method body needs multiple lines it's better to wrap it with `{ }`. Same thing below.